TestApp/ios/TestApp/TestAppNative.m

```diff
@@ -46,4 +58,20 @@ @implementation TestAppNative
   *p = 0;
 }
 
+RCT_EXPORT_METHOD(produceLowMemoryWarning)
+{
+  [_dataArray addObject:[self create256mbRandomNSData]];
+
+  dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 1 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+    [self produceLowMemoryWarning];
+  });
+}
+
+-(NSData*)create256mbRandomNSData {
+  void *bytes = malloc(blockSize);
+  NSData *data = [NSData dataWithBytes:bytes length:blockSize];
+  free(bytes);
```

Comment:
Why do you need free here? 😉

Reply:
[- (instancetype)initWithCapacity:(NSUInteger)capacity;](https://developer.apple.com/documentation/foundation/nsmutabledata/1413350-initwithcapacity?language=objc)

Reply:
initWithCapacity leads to crash without any warnings, so, for now, I just removed 'free' call.